kubernetes/check-pods.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/bash -e
+
+pod_label=$1
+statefulset=$2
```

Comment:
Please add a short description that says what the parameters are and how to run the script.

Reply:
Done.